packages/dd-trace/src/proxy.js

```diff
@@ -58,7 +79,7 @@ class Tracer extends NoopProxy {
       }
 
       if (config.remoteConfig.enabled && !config.isCiVisibility) {
-        const rc = remoteConfig.enable(config)
+        const rc = remoteConfig.enable(config, this)
```

Comment:
here we could pass to RC `this._modules.appsec` instead `this`. That way RC wouldn't need to know where to obtain the appsec module. It'd directly invoke enable or disable.

Reply:
good point, but i feel like we have this habit of passing the tracer instance a bit everywhere, so, up to you :shrug:

Reply:
It's a habit we want to revert ASAP, so I'd use the boy scout rule here if possible.